### ISSUE TITLE
galaxy image ID out of sync

### ISSUE THREAD

**Issue by jbardin** [litl/galaxy#208]
Image ID can get out of sync, with the error:

    ERROR: Pulled image for registry.wovops.net/royal-jelly:latest does not match expected ID. Expected: d219c4901a33: Got: eba640637891

Doesn't seem to resolve itself

**Comment by joeshaw** [comment]
I think this is a race condition triggered when the tag changes mid-deploy.

**Comment by jbardin** [comment]
yeah, I think last time I saw this, it was 2 builders going in parallel, one won the race to galaxy, and the other won the docker push. I'll just have to step through the deploy process to see what we can do.
